pkg/controller/jobs/pod/pod_webhook.go

```diff
@@ -182,6 +185,13 @@ func (w *PodWebhook) Default(ctx context.Context, obj runtime.Object) error {
 
 		gate(&pod.pod)
 
+		if features.Enabled(features.TopologyAwareScheduling) && jobframework.PodSetTopologyRequest(&pod.pod.ObjectMeta) != nil {
+			// The label is added during the creation of pods that request TAS.
```

Comment:
Actually the comments here are kind of obvious, I would prefer to move them to https://github.com/kubernetes-sigs/kueue/blob/e751c8b4b56a1d53dbd9437af93a7043ff9abb49/apis/kueue/v1alpha1/tas_types.go#L41-L59.
For gate and for the label we can say: "For the Pod-based integrations the label is added in webhook during the Pod creation.".

Reply:
Done.